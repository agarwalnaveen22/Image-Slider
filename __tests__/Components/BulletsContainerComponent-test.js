import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import BulletsContainer from '../../Components/BulletsContainerComponent';
import imageData from '../ImageData.json';

describe('test bullet container component', () => {
  const images = imageData;
  it('match snapshot', () => {
    const snap = renderer.create(<BulletsContainer images={images} />).toJSON();
    expect(snap).toMatchSnapshot();
  });
});
