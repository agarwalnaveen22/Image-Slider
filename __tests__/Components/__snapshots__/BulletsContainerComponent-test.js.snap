// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test bullet container component match snapshot 1`] = `
Array [
  <Text
    allowFontScaling={false}
    style={
      Array [
        Object {
          "color": undefined,
          "fontSize": 12,
        },
        Array [
          Object {
            "alignSelf": "center",
            "color": "#000000",
            "fontSize": 15,
            "paddingEnd": 2,
            "paddingStart": 2,
          },
          undefined,
        ],
        Object {
          "fontFamily": "FontAwesome5Free-Regular",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        Object {
          "fontWeight": "400",
        },
      ]
    }
  >
    
  </Text>,
  <Text
    allowFontScaling={false}
    style={
      Array [
        Object {
          "color": undefined,
          "fontSize": 12,
        },
        Array [
          Object {
            "alignSelf": "center",
            "color": "#000000",
            "fontSize": 15,
            "paddingEnd": 2,
            "paddingStart": 2,
          },
          undefined,
        ],
        Object {
          "fontFamily": "FontAwesome5Free-Regular",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        Object {
          "fontWeight": "400",
        },
      ]
    }
  >
    
  </Text>,
  <Text
    allowFontScaling={false}
    style={
      Array [
        Object {
          "color": undefined,
          "fontSize": 12,
        },
        Array [
          Object {
            "alignSelf": "center",
            "color": "#000000",
            "fontSize": 15,
            "paddingEnd": 2,
            "paddingStart": 2,
          },
          undefined,
        ],
        Object {
          "fontFamily": "FontAwesome5Free-Regular",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        Object {
          "fontWeight": "400",
        },
      ]
    }
  >
    
  </Text>,
  <Text
    allowFontScaling={false}
    style={
      Array [
        Object {
          "color": undefined,
          "fontSize": 12,
        },
        Array [
          Object {
            "alignSelf": "center",
            "color": "#000000",
            "fontSize": 15,
            "paddingEnd": 2,
            "paddingStart": 2,
          },
          undefined,
        ],
        Object {
          "fontFamily": "FontAwesome5Free-Regular",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        Object {
          "fontWeight": "400",
        },
      ]
    }
  >
    
  </Text>,
  <Text
    allowFontScaling={false}
    style={
      Array [
        Object {
          "color": undefined,
          "fontSize": 12,
        },
        Array [
          Object {
            "alignSelf": "center",
            "color": "#000000",
            "fontSize": 15,
            "paddingEnd": 2,
            "paddingStart": 2,
          },
          undefined,
        ],
        Object {
          "fontFamily": "FontAwesome5Free-Regular",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        Object {
          "fontWeight": "400",
        },
      ]
    }
  >
    
  </Text>,
  <Text
    allowFontScaling={false}
    style={
      Array [
        Object {
          "color": undefined,
          "fontSize": 12,
        },
        Array [
          Object {
            "alignSelf": "center",
            "color": "#000000",
            "fontSize": 15,
            "paddingEnd": 2,
            "paddingStart": 2,
          },
          undefined,
        ],
        Object {
          "fontFamily": "FontAwesome5Free-Regular",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        Object {
          "fontWeight": "400",
        },
      ]
    }
  >
    
  </Text>,
  <Text
    allowFontScaling={false}
    style={
      Array [
        Object {
          "color": undefined,
          "fontSize": 12,
        },
        Array [
          Object {
            "alignSelf": "center",
            "color": "#000000",
            "fontSize": 15,
            "paddingEnd": 2,
            "paddingStart": 2,
          },
          undefined,
        ],
        Object {
          "fontFamily": "FontAwesome5Free-Regular",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        Object {
          "fontWeight": "400",
        },
      ]
    }
  >
    
  </Text>,
  <Text
    allowFontScaling={false}
    style={
      Array [
        Object {
          "color": undefined,
          "fontSize": 12,
        },
        Array [
          Object {
            "alignSelf": "center",
            "color": "#000000",
            "fontSize": 15,
            "paddingEnd": 2,
            "paddingStart": 2,
          },
          undefined,
        ],
        Object {
          "fontFamily": "FontAwesome5Free-Regular",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        Object {
          "fontWeight": "400",
        },
      ]
    }
  >
    
  </Text>,
  <Text
    allowFontScaling={false}
    style={
      Array [
        Object {
          "color": undefined,
          "fontSize": 12,
        },
        Array [
          Object {
            "alignSelf": "center",
            "color": "#000000",
            "fontSize": 15,
            "paddingEnd": 2,
            "paddingStart": 2,
          },
          undefined,
        ],
        Object {
          "fontFamily": "FontAwesome5Free-Regular",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        Object {
          "fontWeight": "400",
        },
      ]
    }
  >
    
  </Text>,
  <Text
    allowFontScaling={false}
    style={
      Array [
        Object {
          "color": undefined,
          "fontSize": 12,
        },
        Array [
          Object {
            "alignSelf": "center",
            "color": "#000000",
            "fontSize": 15,
            "paddingEnd": 2,
            "paddingStart": 2,
          },
          undefined,
        ],
        Object {
          "fontFamily": "FontAwesome5Free-Regular",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        Object {
          "fontWeight": "400",
        },
      ]
    }
  >
    
  </Text>,
]
`;
